addons/contour/1.10.1/patches/job-image.yaml

```diff
@@ -2,12 +2,19 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
-  name: contour-certgen-v1.10.1
+  name: contour-certgen-v1.10.1-1
   namespace: projectcontour
 spec:
   template:
     spec:
       containers:
       - name: contour
-        image: docker.io/projectcontour/contour:v1.10.1
+        image: projectcontour/contour:v1.10.1
         imagePullPolicy: IfNotPresent
+        command:
+        - contour
+        - certgen
+        - --kube
+        - --incluster
```

Comment:
This removes the `overwrite` flag to prevent any issues of making a new cert while contour pods are already running.